python/lsst/pipe/tasks/colorterms.py

```diff
@@ -166,7 +166,7 @@ def getColorterm(self, filterName, refCatName, doRaise=True):
             ctDictConfig = self.data.get(refCatName)
             if ctDictConfig is None:
                 # try glob expression
-                matchList = [glob for glob in self.data if fnmatch.fnmatch(refCatName, glob)]
+                matchList = [glob for glob in self.data if fnmatch.fnmatch(glob, refCatName)]
```

Comment:
I believe the intent is for the reference catalog names to be able to include metacharacters like "*", so the `refCatName` is actually the glob, whereas the variable named `glob` is the iterator across the elements in `self.data`.  I think this is a case where the variable should be renamed; if it were something like `configName` or even just `x`, this confusion would not occur.